packages/atlas/src/providers/transactionManager/useTransaction.ts

```diff
@@ -100,6 +105,17 @@ export const useTransaction = (): HandleTransactionFn => {
           addBlockAction({ callback: syncCallback, targetBlock: result.block })
         })
 
+        const { data } = await getTransactionStatus({ variables: { transactionHash: result.transactionHash } })
```

Comment:
This needs to happen after `queryNodeSyncPromise` is resolved - this gives us information that QN already processed the block in which our tx was included